shapeshift/web
# feat: bump lib packages to latest major

## Description

This updates lib packages to their latest major following the `CHAIN_NAMESPACE` terminology breaking change in https://github.com/shapeshift/lib/pull/1006 and https://github.com/shapeshift/lib/pull/1007, see the output of `yarn upgrade-interactive` in this diff:

<img width="922" alt="image" src="https://user-images.githubusercontent.com/17035424/188202641-f9b204c9-4378-4abc-a955-22c0ed4282da.png">


## Included changes:

- [`@shapeshiftoss/asset-service` 7.5.0..8.0.1](https://github.com/shapeshift/lib/blob/main/packages/asset-service/CHANGELOG.md)
  - `CHAIN_NAMESPACE` terminology breaking change
- [`@shapeshiftoss/chain-adapters` 7.17.0..9.0.1](https://github.com/shapeshift/lib/blob/main/packages/chain-adapters/CHANGELOG.md)
  - `CHAIN_NAMESPACE` terminology breaking change
  - fix: also test ltc segwit native bip44Params (https://github.com/shapeshift/lib/issues/997) ([8b1192d](https://github.com/shapeshift/lib/commit/8b1192d32e40d7fc4c0d2c8892ae2dc51c40725c))
- [`@shapeshiftoss/investor-foxy` 4.1.0..5.0.1](https://github.com/shapeshift/lib/blob/main/packages/investor-foxy/CHANGELOG.md)
  - `CHAIN_NAMESPACE` terminology breaking change
  -  refactor: tidy usage of bnOrZero (https://github.com/shapeshift/lib/issues/973) ([769ee49](https://github.com/shapeshift/lib/commit/769ee499352df09c6dd1b63467cc6a1ca1bacfaa))
  -  chore: add trailing comma lint rule (https://github.com/shapeshift/lib/issues/975) ([505516c](https://github.com/shapeshift/lib/commit/505516c))
- [`@shapeshiftoss/investor-idle` 1.1.0..2.0.1](https://github.com/shapeshift/lib/blob/main/packages/investor-idle/CHANGELOG.md)
  - `CHAIN_NAMESPACE` terminology breaking change
  -  refactor: tidy usage of bnOrZero (https://github.com/shapeshift/lib/issues/973) ([769ee49](https://github.com/shapeshift/lib/commit/769ee499352df09c6dd1b63467cc6a1ca1bacfaa))
  -  chore: add trailing comma lint rule (https://github.com/shapeshift/lib/issues/975) ([505516c](https://github.com/shapeshift/lib/commit/505516c))
- [`@shapeshiftoss/investor-yearn` 4.1.0..5.0.1](https://github.com/shapeshift/lib/blob/main/packages/investor-yearn/CHANGELOG.md)
  - `CHAIN_NAMESPACE` terminology breaking change
  -  refactor: tidy usage of bnOrZero (https://github.com/shapeshift/lib/issues/973) ([769ee49](https://github.com/shapeshift/lib/commit/769ee499352df09c6dd1b63467cc6a1ca1bacfaa))
  -  chore: add trailing comma lint rule (https://github.com/shapeshift/lib/issues/975) ([505516c](https://github.com/shapeshift/lib/commit/505516c))
- [`@shapeshiftoss/market-service` 6.5.0..7.0.1](https://github.com/shapeshift/lib/blob/main/packages/market-service/CHANGELOG.md)
  - `CHAIN_NAMESPACE` terminology breaking change
  -  chore: add trailing comma lint rule (https://github.com/shapeshift/lib/issues/975) ([505516c](https://github.com/shapeshift/lib/commit/505516c))
- [`@shapeshiftoss/swapper` 9.17.1..11.0.1](https://github.com/shapeshift/lib/blob/main/packages/swapper/CHANGELOG.md)
  - `CHAIN_NAMESPACE` terminology breaking change
  -   fix: osmo: calculate finish tx off second tx in transaction (https://github.com/shapeshift/lib/issues/990) ([a879550](https://github.com/shapeshift/lib/commit/a879550))
- [`@shapeshiftoss/unchained-client` 9.9.0..10.0.1](https://github.com/shapeshift/lib/blob/main/packages/unchained-client/CHANGELOG.md)
  - `CHAIN_NAMESPACE` terminology breaking change

## Notice




## Pull Request Type


## Issue (if applicable)

N/A - relates to mutli-account M2 https://github.com/shapeshift/web/milestone/31

## Risk



## Testing

- General regression testing
- Ensure there are no browser console weirdnesses with the updated packages
- Ensure Swapper Osmosis Txs show a confirmed state after completion on-chain (https://github.com/shapeshift/lib/pull/990)

<img width="390" alt="image" src="https://user-images.githubusercontent.com/17035424/188207691-616c3190-308c-4060-92bf-b4bd10c719ec.png">

### Engineering

- Refer to the top-level testing notes
- CI is happy

### Operations

- Refer to the top-level testing notes

## Screenshots (if applicable)
